src/sentry/static/sentry/app/views/eventsV2/resultsChart.tsx

```diff
@@ -76,7 +76,7 @@ class ResultsChart extends React.Component<ResultsChartProps> {
               end={end}
               period={globalSelection.statsPeriod}
               disablePrevious={display !== DisplayModes.PREVIOUS}
-              disableReleases={display !== DisplayModes.RELEASES}
+              disableReleases={isDaily}
```

Comment:
Should releases be disabled in previous period mode? It may be confusing to show releases on top of the previous period

Reply:
Yes, I having releases on top of two different timelines is confusing as you can't show releases from both timelines.